lib/diagnostic.js

```diff
@@ -1,21 +1,38 @@
 'use strict'
 
 // Reuse functions wherever possible.
-
+// create a string
+const paragraph = 'Hello Hello my name is Nikki'
 // 1. Write a function that takes a string argument
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+
+// split the words
+const words = paragraph.split(' ')
+// words => ["Hello", "Hello", my", "name", "is", "Nikki"]
+
+const normalizeWords = function () {
+	for (let i = 0; i < words.length; i++) {
+	  normalizedWords[i] = words[i].toUpperCase()
```

Comment:
Very close, but you need to have this function accept a string and then split that string into an array using `.split()`